test/functional/pdo_sqlsrv/pdostatement_closeCursor.phpt

```diff
@@ -1,28 +1,27 @@
 --TEST--
 Test PDOStatement::closeCursor method.
 --SKIPIF--
-<?php require('skipif.inc'); ?>
+<?php require('skipif_mid-refactor.inc'); ?>
 --FILE--
 <?php
-require_once 'MsCommon.inc';
+require_once("MsCommon_mid-refactor.inc");
+require_once("MsData_PDO_AllTypes.inc");
 
-try{
+try {
     $db = connect();
-    global $table2;
-    $stmt = $db->prepare("SELECT * FROM " . $table2);
+    $tbname = "PDO_AllTypes";
+    createAndInsertTableAllTypes($db, $tbname);
+    $stmt = $db->prepare("SELECT * FROM $tbname");
     $stmt->execute();
     $result = $stmt->fetch();
-    $stmt->closeCursor();
-    echo "Test complete!";
-}
-catch ( PDOException $e)
-{
+
```

Comment:
probably not very important, but why did you remove closeCursor()?

Reply:
I think back then when we didn't know the driver well enough, we always get confused between $stmt->closeCursor(), $stmt=null and unset($stmt). I think what the test is trying to do is to clean the statement, which I did using unset($stmt).